pkg/config/autodiscover/autodiscover_targets.go

```diff
@@ -301,6 +303,7 @@ func buildOperatorFromCSVResource(csv *CSVResource, istest bool) (op configsecti
 		op.SubscriptionName = subscriptionName[0]
 	}
 	if !istest {
+		op.BundleImage, op.IndexImage = getBundleAndIndexImage(op.Name, op.Namespace)
```

Comment:
Check error variable right after this call:
```
op.BundleImage, op.IndexImage, err = getBundleAndIndexImage(op.Name, op.Namespace)
if err != nil {
    log.Errorf("Failed to get operator bundle and index image for csv % (ns %s), error: %s", op.Name, op.Namespace, err)
}
```

Reply:
done